### ISSUE TITLE
could not disable smart typography

### ISSUE THREAD

**Issue by simonmichael** [jgm/pandoc#4788]
I'm getting unwanted [smart typography](http://pandoc.org/MANUAL.html#typography) and could not figure out how to turn it off.
```
$ pandoc --version
pandoc 2.2.1
Compiled with pandoc-types 1.17.4.2, texmath 0.11, skylighting 0.7.1
Default user data directory: /Users/simon/.pandoc
Copyright (C) 2006-2018 John MacFarlane
Web:  http://pandoc.org
This is free software; see the source for copying conditions.
There is no warranty, not even for merchantability or fitness
for a particular purpose.
$ echo "..." | pandoc -f org -t html
<p>…</p>
$ echo "..." | pandoc -f org-smart -t html-smart
<p>…</p>
```
Seen on both macos and ubuntu.

**Comment by simonmichael** [comment]
I can turn it off when reading markdown, though; but not org.
```
$ echo "..." | pandoc -f markdown-smart
<p>...</p>
```
I think this is saying it's supposed to be off by default for the org reader:
```
$ pandoc --list-extensions=org
-abbreviations
-all_symbols_escapable
-amuse
-angle_brackets_escapable
-ascii_identifiers
+auto_identifiers
-autolink_bare_uris
-backtick_code_blocks
-blank_before_blockquote
-blank_before_header
-bracketed_spans
+citations
-compact_definition_lists
-definition_lists
-east_asian_line_breaks
-emoji
-empty_paragraphs
-epub_html_exts
-escaped_line_breaks
-example_lists
-fancy_lists
-fenced_code_attributes
-fenced_code_blocks
-fenced_divs
-footnotes
-four_space_rule
-gfm_auto_identifiers
-grid_tables
-hard_line_breaks
-header_attributes
-ignore_line_breaks
-implicit_figures
-implicit_header_references
-inline_code_attributes
-inline_notes
-intraword_underscores
-latex_macros
-line_blocks
-link_attributes
-lists_without_preceding_blankline
-literate_haskell
-markdown_attribute
-markdown_in_html_blocks
-mmd_header_identifiers
-mmd_link_attributes
-mmd_title_block
-multiline_tables
-native_divs
-native_spans
-ntb
-old_dashes
-pandoc_title_block
-pipe_tables
-raw_attribute
-raw_html
-raw_tex
-shortcut_reference_links
-simple_tables
-smart
-space_in_atx_header
-spaced_reference_links
-startnum
-strikeout
-subscript
-superscript
-styles
-table_captions
-tex_math_dollars
-tex_math_double_backslash
-tex_math_single_backslash
-yaml_metadata_block
```

**Comment by tarleb** [comment]
Workaround (untested):

`echo "#+OPTIONS: -:nil\n..." | pandoc -f org`

That's because we are not only checking the pandoc setting, but also the org-mode options. The latter have this feature turned in by default.

**Comment by tarleb** [comment]
The shell won't interpret the `\n` as a newline when it's in single quotes. Using double quotes should work.

I'm thinking if we should change the current behavior, but we are in a bit of a dilemma here. Org mode has special chars (turning `...` into `…` and `--` into `–`) enabled and smart quotes disabled by default. This is the behavior we need to match with pandoc, while also respecting the *smart* extension. We can either keep the current behavior, or invert it and have *smart* enabled by default. Then either of `-smart` *and* `#+OPTIONS: -:nil` would turn off special characters, but `+smart` would no longer be enough to enable smart quotes. So the question is what's more common, tuning special characters off or smart quotes on? What do you think?

**Comment by simonmichael** [comment]
Ah, thanks! You have to use printf for `\n`. `printf "#+OPTIONS: -:nil\n..." | pandoc -f org`

I think I've been using pandoc pretty frequently since its early days. I'm usually creating online documentation, plumbing docs through multiple tools, and moving snippets from docs to pastes to terminal or code, and back. For my usage, smart quotes and translation of character sequences is simply more trouble than it's worth and I never want it enabled. So far.

**Comment by tarleb** [comment]
Thanks. I'll have to think about this some more, but will close the issue for now.

**Comment by mb21** [comment]
@tarleb but maybe add a sentence to the Manual?

**Comment by tarleb** [comment]
Yes, that'd be good. I keep meaning to write proper documentation for pandoc's org features, see #4387. I've added this to the list there.

**Comment by simonmichael** [comment]
This bit me again just now. (Trying to convert an org-format changelog to plain text for emailing, wondering why are all my --options converted to –options ...)

**Comment by simonmichael** [comment]
> I'm thinking if we should change the current behavior, but we are in a bit of a dilemma here. Org mode has special chars (turning `...` into `…` and `--` into `–`) enabled and smart quotes disabled by default. This is the behavior we need to match with pandoc, while also respecting the _smart_ extension. We can either keep the current behavior, or invert it and have _smart_ enabled by default. Then either of `-smart` _and_ `#+OPTIONS: -:nil` would turn off special characters, but `+smart` would no longer be enough to enable smart quotes. So the question is: what's the more common use-case, tuning special characters off or smart quotes on? What do you think?

@tarleb if I understand you right, the pandoc org (reader ? writer ?) has a different default for `...` and for `'` conversion ? This seems like a recipe for confusion, and contrary to http://pandoc.org/MANUAL.html#typography which implies `smart` covers them all.

**Comment by jgm** [comment]
Simon Michael <notifications@github.com> writes:

>> I'm thinking if we should change the current behavior, but we are in a bit of a dilemma here. Org mode has special chars (turning `...` into `…` and `--` into `–`) enabled and smart quotes disabled by default. This is the behavior we need to match with pandoc, while also respecting the _smart_ extension. We can either keep the current behavior, or invert it and have _smart_ enabled by default. Then either of `-smart` _and_ `#+OPTIONS: -:nil` would turn off special characters, but `+smart` would no longer be enough to enable smart quotes. So the question is: what's the more common use-case, tuning special characters off or smart quotes on? What do you think?
>
> @tarleb if I understand you right, the pandoc org (reader ? writer ?) has a different default for `...` and for `'` conversion ? This seems like a recipe for confusion, and contrary to http://pandoc.org/MANUAL.html#typography which implies `smart` covers them all. 

We try to be consistent with org's own defaults.
Otherwise it is confusing to the user, who gets a
different result form org export and pandoc.

The problem is that org doesn't have a single `smart`;
it has two switches, and they have different values by
default.

We can't match this unless we add an additional
extension that just affects org mode.  Maybe we
should do that. @tarleb?

**Comment by simonmichael** [comment]
Oh drat. Well then +1 for some way to control it from the command line, rather than the `#+OPTIONS` workaround above; and/or, to have it clarified in docs. If I can help, let me know.

**Comment by jgm** [comment]
Simon Michael <notifications@github.com> writes:

> Oh drat. Well then +1 for some way to control it from the command line, rather than the `#+OPTIONS` workaround above; and/or, to have it clarified in docs. If I can help, let me know.


@tarleb I'm curious to hear your thoughts on what the
best approach would be.
